test/consts/feature_gates.go

```diff
@@ -6,4 +6,8 @@ const (
 	// that are innocuous, or otherwise don't actually get triggered unless the
 	// user takes further action.
 	DefaultFeatureGates = "GatewayAlpha=true"
+
+	// ConformanceTestsFeatureGates is the set of feature gates to be used when running
+	// conformance tests.
+	ConformanceTestsFeatureGates = "GatewayAlpha=true,ExpressionRoutes=true"
```

Comment:
Given the expression router is still an alpha feature in KIC (it's default-off), would that make sense to run Gateway API conformance tests for both traditional and expression routers? We could have separate `SkipTests` for them, WDYT?

Reply:
See commit https://github.com/Kong/kubernetes-ingress-controller/pull/4359/commits/77f218abc82086120c870fdf1f104c30fe9beb7d